src/schema/v2/homeView/sections/ExploreByCategories.ts

```diff
@@ -0,0 +1,64 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+
+export const ExploreByCategories: HomeViewSection = {
+  id: "home-view-section-explore-by-categories",
+  type: HomeViewSectionTypeNames.HomeViewSectionMarketingCollectionCategories,
+  // TODO: add right tracking
+  contextModule: ContextModule.categoryRail,
```

Comment:
**thought (non-blocking):** FYI this was very recently (after the KS!) [made optional](https://github.com/artsy/metaphysics/pull/6055/files), so you can also comment/omit this line until you have the correct Cohesion settings ready.

Reply:
ah I tried to remove these before rebasing my branch and I couldn't, will remove it then 👍